webapp/src/mutator.ts

```diff
@@ -1196,14 +1196,14 @@ class Mutator {
         return octoClient.importFullArchive(file)
     }
 
-    async importFullArchiveTrello(file: File, signupToken: string): Promise<Record<string, string>[]> {
-        const fileContents = await new Response(file.stream()).text();
+    async importFullArchiveTrello(file: File, signupToken: string): Promise<Array<Record<string, string>>> {
+        const fileContents = await new Response(file.stream()).text()
         const input = JSON.parse(fileContents) as Trello
 
         const memberIdMap = new Map<string, string>()
-        const newUserInfo: Record<string, string>[] = []
+        const newUserInfo: Array<Record<string, string>> = []
```

Comment:
not sure why but ok

Reply:
eslint typescript plugin (?) did it for me on save
agreed to trust it